package.json

```diff
@@ -113,8 +113,9 @@
     "release": "standard-version",
     "tasks:keycloak": "node dist/tasks/keycloak/keycloak.js",
     "tasks:keycloak-dev": "ts-node-dev ./src/tasks/keycloak/keycloak.ts",
-    "tasks:gitea": "node dist/tasks/gitea/gitea.js",
+    "tasks:gitea": "node dist/tasks/gitea/gitea.js && node dist/tasks/gitea/gitea-drone-oauth.js",
```

Comment:
does it make sense to create `tasks:gitea-drone-auth` and to run it as a separate job ?

Reply:
I think not